app/interfaces/api/entities/search_result.rb

```diff
@@ -98,6 +106,10 @@ def interim_disbursements
       def risk_based_bills
         ((risk_based_class_letter && contains_risk_based_fee).eql?(true) && is_submitted?).to_i
       end
+
+      def injection_errored
+        object.injection_errors.blank? ? 0 : 1
```

Comment:
I'm pretty sure you could use `object.injection_errors.blank?.to_i` to match the other filters

Reply:
:thumbsup: will give it a go

Reply:
yep, looks good
```ruby
[121] Pry:Claim-for-Crown-Court-Defence(main)> 'whatevs'.present?.to_i
=> 1
[122] Pry:Claim-for-Crown-Court-Defence(main)> ''.present?.to_i
=> 0
[123] Pry:Claim-for-Crown-Court-Defence(main)> nil.present?.to_i
=> 0
[124] Pry:Claim-for-Crown-Court-Defence(main)> false.present?.to_i
=> 0

```